tests/Api/ContactsTest.php

```diff
@@ -163,6 +164,25 @@ public function testCreateGetAndDelete()
         $this->assertErrors($response);
     }
 
+    public function testMergingDuplicateContacts()
+    {
+        // Create contact A
+        $response = $this->api->create($this->testPayload);
+        $this->assertPayload($response);
+        $contactA = $response[$this->api->itemName()];
+
+        // Create contact B
+        $response = $this->api->create($this->testPayload);
+        $this->assertPayload($response);
+        $contactBc = $response[$this->api->itemName()];
+
+        $response = $this->api->edit($contactBc['id'], $this->testPayload);
+        $this->assertPayload($response);
+        $contactBe = $response[$this->api->itemName()];
+
+        $this->assertErrors($contactBe);
```

Comment:
This test will check if the contacts are created and updated fine, but we have another test to do that. This test should check if the contacts were merged somehow. I was thinking something about

```
$this->assertSame($contactA['id'], $contactB['id']);
```
If it should work like that.

I don't know if the "edit" part is important. But I added it there since the PR description said it works only for updates.

`$this->assertErrors($contactBe);` won't do anything. It is part of the `assertPayload` method and it expects `$response` array, not just the contact. So you can remove this line.